### DISCUSSION TITLE
Achieving complete transparency with MAUI & TransparentTintBackdrop

### DISCUSSION THREAD

**Discussion by CodyLyttle** [microsoft/microsoft-ui-xaml#9157] [Q&A] [answered]
I'm using the following code to create an entirely transparent background `new TransparentTintBackdrop(Color.FromArgb(0, 0,0,0));` Unfortunately, this leaves me with a partially transparent gray background.
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/18041702/24b8911a-1cc4-485d-87f9-d91950f16c50)

What I've tried:
Modifying Background & BackgroundColor properties in AppShell.xaml.
Modifying Background & BackgroundColor properties in MainPage.xaml.
Removing all default Styles.

Is there a solution or workaround, or is this an unavoidable feature of MAUI?

**Comment by ghost1372** [comment]
In winui 3 you should use p/invoke methods

**Reply by CodyLyttle** [reply]
Just to clarify - are you suggesting I drop TransparentTintBackdrop in favour of p/invoke window styling, or rather use a combination of the two?

**Reply by ghost1372** [reply, marked as answer]
I have not used TransparentTintBackdrop  and I do not know about it.
But according to my latest information, you should use the native methods.
there is a lib called WinUIEx which has a transparent window:
https://github.com/dotMorten/WinUIEx

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/9213496/430da589-05cf-4254-a8a6-acac9978bce8)

**Reply by CodyLyttle** [reply]
I actually intended to start this discussion on the WinUIEx project that you've just linked. Not sure how I made that mistake, my apologies.
